### PR TITLE
firmware_configurations/klipper: Fysetc S6 V2.1

### PR DESCRIPTION
**Pull request by clee**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [ ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

V0.1 example config for Fysetc S6 V2.1, along with separate board pin aliases for other printers.

#### How was it tested? 

I have not tested this specific config with my V0.1, but the pins are all taken from my working config. This needs testing and verification from someone else.

#### Any background context you want to provide?

The S6 supports up to six stepper drivers, which means it's usable for V0.1, Trident, Tri-Zero, and Salad Fork.

### CODE REVIEW

File: firmware_configurations/README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ like so:
 | RealDeuce | [MKS Makerbase Monster8 v1.0_003](./klipper/RealDeuce/MKS-Makerbase/Monster8_v1.0_003/) | Configuration and instructions for installing an MKS Makerbase Monster8 board | :x: | :x: | :heavy_check_mark: | :x: | :x: |
 | TechnoPhreak | [BTT SKR v2 for V2.4](./klipper/TechnoPhreak/BTT_SKR_v2) | Example configuration, wiring diagram and special instructions for the BTT SKR v2 controllers | :grey_question: | :grey_question: | :heavy_check_mark: | :grey_question: |
 | redelman | [Fysetc Spider 2.2 for Voron Trident](./klipper/redelman/Spider_2.2_Voron_Trident) | Example configuration and special instructions for the Fysetc Spider 2.2 for VT | :x: | :x: | :x: | :x: | :heavy_check_mark: |
+| clee | [Fysetc S6 V2.1 for V0.1](./klipper/clee/fysetc_s6) | Example configuration and board pin aliases for the Fysetc S6 V2.1 for V0.1 | :heavy_check_mark: | :x: | :x: | :x: | :x: |
```
Comment (cbc02009) [inline firmware_configurations/README.md:38]: This needs to be in alphabetical order

File: firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+#####################################################################
+#   Bed Heater
+#####################################################################
+
+[heater_bed]
+heater_pin: BED_OUT
+##  Validate the following thermistor type to make sure it is correct
+##  See https://www.klipper3d.org/Config_Reference.html#common-thermistors for additional options
+#sensor_type: NTC 100K MGB18-104F39050L32                           # For Keenovo, verify yours
```
Comment (cbc02009) [inline firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg:201]: according to Eddie, keenovo says we should be using Generic 3950 for their heaters. Have you heard differently?

File: firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+   G1 E30 F300                    ; load
+   G1 E15 F150                    ; prime nozzle with filament
+   M82                            ; set extruder to absolute
+    
+[gcode_macro UNLOAD_FILAMENT]
+gcode:
+   M83                            ; set extruder to relative
+   G1 E10 F300                    ; extrude a little to soften tip
+   G1 E-40 F1800                  ; retract some, but not too much or it will jam
+   M82                            ; set extruder to absolute
```
Comment (cbc02009) [inline firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg:323]: add newline to end of file

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg (hunk 2 above)
Comment (clee) [reply firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg:201]: I started with the SKR Pico V0.1 config file as my template: <https://github.com/VoronDesign/Voron-0/blob/Voron0.1/Firmware/skr-pico-v1.0.cfg> which is where that came from.

File: firmware_configurations/README.md (hunk 1 above)
Comment (clee) [reply firmware_configurations/README.md]: Fixed!

File: firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg (hunk 3 above)
Comment (clee) [reply firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg]: Fixed

File: firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg (hunk 2 above)
Comment (clee) [reply firmware_configurations/klipper/clee/fysetc_s6/v0-fysetc_s6_v2.1.cfg]: Updated to `Generic 3950`.

**Comment by KurioHonoo** [comment]
Based on personal experience the X and Y endstop need to be set to pullup and reversed polarity,  `^!`

**Comment by cbc02009** [comment]
I didn't believe it, but there really isn't anything on those endstop pins. No pull ups, no protection, no filtering...

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**